stablehlo/reference/Ops.cpp

```diff
@@ -90,6 +90,22 @@ Tensor eval(ReshapeOp op, const Tensor &operand) {
   return result;
 }
 
+Tensor eval(MaxOp op, const Tensor &lhs, const Tensor &rhs) {
+  Tensor result(op.getType());
+  for (auto i = 0; i < lhs.getNumElements(); ++i) {
```

Comment:
Update this to use IndexSpaceIterator here and below.

Reply:
Good catch. Thanks!